pkg/server/server.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/grafana/grafana/pkg/login/social"
 	"github.com/grafana/grafana/pkg/middleware"
 	_ "github.com/grafana/grafana/pkg/plugins"
```

Comment:
Could this be removed, assuming that `manager` imports its models?

Reply:
@sakjur good catch, that one is no longer necessary. Thanks!